pkg/controllers/pod.go

```diff
@@ -140,7 +140,7 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 
 	ctx, podScheduledOnMatchingNode, err := r.podScheduledOnMatchingNode(ctx, req.Namespace, pod, podImages)
 	if err != nil {
-		return ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, err
+		return ctrl.Result{}, err
```

Comment:
Naive q: 
Why not requeing this request in case of podScheduledOnMatchingNode() encounters and returns an error? 
One of the possible error causes is that it cannot list Archs from the registry: 
https://github.com/adevinta/noe/blob/218b012eac6ce1f0386be610d09bc5fcc390b263/pkg/controllers/pod.go#L196 
https://github.com/adevinta/noe/blob/218b012eac6ce1f0386be610d09bc5fcc390b263/pkg/registry/registry.go#L603-L606

Reply:
Returning an error, the controller will log the error as such and requeue with an exponential backoff.

The alternative you propose, if I understand correctly, is to return `ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil`, right?

Reply:
yes. if the default behaviour is already requeue+backoff (exponential in that case) it works for me, as recovering from a potential external error it's addressed already.